app/models/goal.py

```diff
@@ -1,5 +1,31 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
 from ..db import db
+from flask import abort, make_response
 
 class Goal(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
```

Comment:
Consider whether the this column for `Goal` should be nullable. It feels unconventional to allow someone to create a goal that gets saved to the DB without a title.